src/organisation/components/pba-numbers-form/pba-numbers-form.component.ts

```diff
@@ -0,0 +1,133 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { AbstractControl, FormArray, FormBuilder, FormControl, FormGroup, ValidatorFn, Validators } from '@angular/forms';
+import { RxwebValidators } from '@rxweb/reactive-form-validators';
+import { UpdatePbaNumbers } from 'src/organisation/models/update-pba-numbers.model';
+
+@Component({
+  selector: 'app-prd-pba-numbers-form-component',
+  templateUrl: './pba-numbers-form.component.html',
+})
+export class PbaNumbersFormComponent implements OnInit {
+
+  public readonly title = 'Add or remove PBA accounts';
```

Comment:
If this is `readonly` and isn't changed in the constructor, should it be `static`?